records_mover/db/loader.py

```diff
@@ -42,6 +42,9 @@ def can_load_this_format(self, source_records_format: BaseRecordsFormat) -> bool
         method"""
         ...
 
+    def temporary_loadable_directory_scheme(self) -> str:
+        return 'file'
```

Comment:
As a refresher:

* DBDriver abstract classes: This is a wrapper on top of SQLAlchemy's database dialects that describe how to load/unload data and also provide some extra information about things like the numeric ranges of types.
* Loader/Unloader abstract classes: These are the interfaces extended for each database to show how to do bulk imports/exports.
* source/target: sources and targets are things like files on disk, directories in S3, tables in databases, etc - you move data with Records Mover from a source to a target.
* Source/Target abstract classes: these are abstract interfaces that show what capabilities a given source or target have.  The table source and table target of course use the DBDriver interface extensively to know what capabilities a specific database have.
* move() algorithm - uses Source and Target objects and inspect to find out which particular Source/Target abstract classes are implemented to figure out the fastest way to move data from a source to a target.

So this is an additional exported bit of info on the Loader interface that helps us know that if we had to create a temporary directory to load from, where it would be created.  We'll see where that's used later on in the PR here.